EIPS/peerdas.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
```suggestion
---
eip: 7594
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.